packages/client/src/__tests__/integration/happy/composites-mongo/count/required.ts

```diff
@@ -0,0 +1,59 @@
+import { getTestClient } from '../../../../../utils/getTestClient'
+
+const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
+
+let PrismaClient, prisma
+
+const id = '9ccccccccccccccccccccccc'
+
+/**
+ * Test count operations on required composite fields
+ */
+describeIf(!process.env.TEST_SKIP_MONGODB)('count > required', () => {
```

Comment:
```suggestion
describeIf(!process.env.TEST_SKIP_MONGODB)('count: required', () => {
```
Nit: A bit weird as it reads as an operator

Reply:
OK, I'll fix that in a followup PR after this is merged.